addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -65,24 +65,23 @@
 
 <t t-name="website.DeletePageDialog">
     <WebsiteDialog title.translate="Delete Page" close="props.close">
-        <p t-if="props.resIds.length > 1">Are you sure you want to delete those pages?</p>
-        <p t-else="">Are you sure you want to delete this page?</p>
-        <p t-if="props.hasNewPageTemplate" class="text-warning">
-            <t t-if="props.resIds.length > 1">Some of these pages are used as new page templates.</t>
-            <t t-else="">This page is used as a new page template.</t>
+        <p t-if="props.resIds.length > 1">Ready to make those web pages disappear into thin air? Are you sure?\n It will be gone forever!\n
+            <span class="text-warning">But before deleting them, make sure you update all links referring to them. That way, your customers won't bump into the not-so-famous 404 error page.</span>
+        </p>
+        <p t-else="">Ready to make this web page disappear into thin air? Are you sure?\n It will be gone forever!\n
+            <span class="text-warning">But before deleting it, make sure you update all links referring to it. That way, your customers won't bump into the not-so-famous 404 error page.</span>
```

Comment:
```suggestion
        <p t-if="props.resIds.length > 1">Ready to make these web pages disappear into thin air? Are you sure?\n They will be gone forever!\n
            <span class="text-warning">But before deleting, make sure you update all links referring to them. That way, your customers won't bump into the not-so-famous 404 error page.</span>
        </p>
        <p t-else="">Ready to make this web page disappear into thin air? Are you sure?\n It will be gone forever!\n
            <span class="text-warning">But before deleting, make sure you update all links referring to it. That way, your customers won't bump into the not-so-famous 404 error page.</span>
```